2/Dockerfile.rhel7

```diff
@@ -41,6 +41,7 @@ RUN yum-config-manager --disable epel >/dev/null || : && \
     yum install -y $INSTALL_PKGS && \
     rpm -V $INSTALL_PKGS && \
     yum clean all  && \
+    curl -fL https://github.com/krallin/tini/releases/download/v0.16.1/tini-static -o /bin/tini && chmod +x /usr/bin/tini && \
```

Comment:
can't do this in rhel.  is it packaged?

Reply:
i could not find a repo to access via yum/dnf, which is why I went this way, but just revisited the release page and it does in fact have an RPM to download

i'll switch to that